gulpfile.js

```diff
@@ -143,6 +145,14 @@ gulp.task('build', ['resources', 'html'], function() {
 
 gulp.task('clean', del.bind(null, ['dist']));
 
-gulp.task('default', ['clean'], function() {
+gulp.task('lint', function() {
+  return gulp.src('app/**/*.js')
+    .pipe(eslint())
+    .pipe(eslint.format());
+  // // TODO: Uncomment when linting looks good
+  // .pipe(eslint.failAfterError());
```

Comment:
Ideally we should fail on linting errors, but I wanted to give you guys a chance to make the fixes first.